commands/python/requirements.txt

```diff
@@ -3,4 +3,4 @@ boto3==1.7.33
 cached-property==1.3.0
 colorlog==2.10.0
 jmespath==0.9.2
-kubernetes==5.0.0
+kubernetes==10.0.1
```

Comment:
Needed to upgrade this because alpine is past python3.6, and the old kubernetes lib assigns to the async keyword

Reply:
will this have to be coordinated with services that use aladdin + kubernetes? (like `loyalty`)

Reply:
alternatively, can we pin to older alpine?

Reply:
older alpine didn't have the right go version to install aws-iam-authenticator